templates/parachain/zombienet/tests/smoke.rs

```diff
@@ -0,0 +1,49 @@
+//! This test is setup to run with the `native` provider and needs these binaries in your PATH
+//! `polkadot`, `polkadot-prepare-worker`, `polkadot-execute-worker`, `parachain-template-node`.
+//! You can follow these steps to compile and export the binaries:
+//! `cargo build --release -features fast-runtime --bin polkadot --bin polkadot-execute-worker --bin
+//! polkadot-prepare-worker`
+//! `cargo build --package parachain-template-node --release`
+//! `export PATH=<path-to-polkadot-sdk-repo>/target/release:$PATH
+//!
+//! The you can run the test with
+//! `cargo test -p parachain-template-zombienet`
```

Comment:
So if someone is running `cargo test` in the repo, it will fail. Then they will be confused and at some point maybe find these docs. TBH, this is not any nice experience. Generally for running these tests it is not a nice experience. 

I still don't get the need for writing them in Rust. This just makes it even more complicated than it already is.

Reply:
Hi @bkchr, thanks for the feedback. I'm agree that the _dx_ needs to improve and isn't in the final shape. Wdyt about make this test feature gated and even if you pass the feature run some sanity check to ensure the preconditions (e.g the binaries) are availables and if not print a nice guide about the requirements?